### ISSUE TITLE
SX M26C camera crashes PHD2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenPHDGuiding/phd2#420]
```
using the native Starlight Xpress SXV driver
when exposure looping starts with the M26C the application crashes

```

Original issue reported on code.google.com by `andy.gal...@gmail.com` on 11 Apr 2015 at 2:09

**Comment by agalasso** [comment]
15:15:19.393 00.016 3528 SX Camera: SX-M26
15:15:19.393 00.000 3528 SX Camera Params: 3900 x 1308 (reported as 3900 x 1308) PixSz: 6.05 x 6.05; #Pix: 5101200 Array color type: 4095,0 Interlaced: 1 Model: 90, Subype: 26, Porch: 45,10 8,8 Extras: 21
15:15:19.393 00.000 3528 Connected Camera:SX-M26

**Comment by agalasso** [comment]
Problem was caused by the SquarePixels option not handling the case where pixels were already square. Fixed with commit 252714f04a456c351b553015562cf3fa6eacac3a
